doc/README.hive

```diff
@@ -0,0 +1,155 @@
+   downloading and patching easier.
+
+1) Select an appropriate submission script for running your job.  You
+   can find them in the directory submission-scripts/, with Slurm
+   Sbatch scripts using srun in script-sbatch-srun, Moab Msub+Slurm
+   scripts using srun in script-msub-slurm-srun, Moab Msub+Torque
+   scripts using pdsh in script-msub-torque-pdsh, and LSF scripts
+   using mpirun in script-lsf-mpirun.
+
+   You'll likely want to start with the base hive w/ hdfs script
+   (e.g. magpie.sbatch-srun-hive-with-hdfs) for your
+   scheduler/resource manager.  If you wish to configure more, you can
+   choose to start with the base script (e.g. magpie.sbatch-srun)
+   which contains all configuration options.
+
+2) Setup your job essentials at the top of the submission script.  As
+   an example, the following are the essentials for running with Moab.
+
+   #MSUB -l nodes : Set how many nodes you want in your job
+
+   #MSUB -l walltime : Set the time for this job to run
+
+   #MSUB -l partition : Set the job partition
+
+   #MSUB -q <my batch queue> : Set to batch queue
+
+   MOAB_JOBNAME : Set your job name.
+
+   MAGPIE_SCRIPTS_HOME : Set where your scripts are
+
+   MAGPIE_LOCAL_DIR : For scratch space files
+
+   MAGPIE_JOB_TYPE : This should be set to 'hive'
+
+   JAVA_HOME : B/c you need to ...
+
+3) Setup the essentials for Hive.
+
+   HIVE_SETUP : Set to yes
+
+   HIVE_VERSION : Set appropriately.
+
+   HIVE_HOME : Where your Hive code is.  Typically in an NFS
+   mount.
+
+   HIVE_LOCAL_DIR : A small place for conf files and log files local
+   to each node.  Typically /tmp directory.
+
+   ANT_LIB : (optional) Path to the Apache Ant for Hive web interface
+   
+   HIVE_METASTORE_DIR : Path for Hive to use for the Hive Metastore
+
+   HIVE_DEFAULT_METASTORE_DIR : Default path if HIVE_METASTORE_DIR doesn't exist
+
+   HIVE_DB_NAME : Name of the database for this stack
+
+   POSTGRES_HOME : Where PostgreSQL binary lives
+
+   PGPORT : The port that PostgreSQL will listen for connections on. 11115 works well
+ 
+   PGDATA : The path for PostgreSQL to use for its tables
+
+
+4) Select how your job will run by setting HIVE_MODE.  The first time
+   you'll probably want to run w/ 'performanceeval' mode just to try
+   things out and make sure things look setup correctly.
+
+   After this, you may want to run with 'interactive' mode to play
+   around and figure things out.  In the job output you will see
+   output similar to the following:
+
+      ssh node70
+      setenv JAVA_HOME "/usr/lib/jvm/jre-1.7.0-oracle.x86_64/"
+      setenv HIVE_HOME "/home/username/apache-hive-2.3.0-bin"
+      setenv HIVE_CONF_DIR "/tmp/username/hive/ajobtime/1174573/conf"
+
+   These instructions will inform you how to login to the master node
+   of your allocation and how to initialize your session.  Once in
+   your session.  You can do as you please.  For example, you can
+   interact with the Hive database to start (bin/beeline ).  There
```

Comment:
English error.  not your fault, as it was in all the other docs (fixed upstream now).  "Once in your session, you can do as you please."